### DISCUSSION TITLE
plantuml.requarks.io offline

### DISCUSSION THREAD

**Discussion by ymoreiratiti** [requarks/wiki#5833] [Error / Bug Report] [answered]
Hello! 

Is https://plantuml.requarks.io unavailable?

![image](https://user-images.githubusercontent.com/5731429/201389178-644170ef-6ed9-417f-9445-1c449e234041.png)

![image](https://user-images.githubusercontent.com/5731429/201389159-013723ef-507d-4ff2-aa71-905de1220447.png)

**Comment by NGPixel** [comment, marked as answer]
I restarted the service. It should be up again now.
